daemon/containerd/progress.go

```diff
@@ -189,6 +189,8 @@ func (p *pullProgress) UpdateProgress(ctx context.Context, ongoing *jobs, out pr
 		}
 
 		switch sn.Kind {
+		default:
+			// Ignore unknown snapshot kinds
```

Comment:
Oh! I left a comment on the commit that introduced this switch; instead of `default`, for this one it's probably better to be explicit (as it's an exhaustive number of options).

That change should be in the same commit.